### ISSUE TITLE
Remounting the filesystem RO fails in some rare cases

### ISSUE THREAD

**Issue by valr** [valr/initsystem#1]
In some cases, the root FS cannot be remounted RO during shutdown/reboot.
I have the impression it's after the 'glibc' package is upgraded, but can't confirm.

**Comment by valr** [comment]
Looking at the .install file of the glibc package, there is this command executed:
`ldconfig -r .`

And as per man page:
`       -r root : Change to and use root as the root directory.
`

**Comment by valr** [comment]
Root cause has been identified, lsof shows this for the init process:
```
   1   │ COMMAND    PID  TID TASKCMD      USER   FD      TYPE             DEVICE  SIZE/OFF       NODE NAME
   2   │ init-og      1                   root  cwd       DIR                8,1      4096          2 /
   3   │ init-og      1                   root  rtd       DIR                8,1      4096          2 /
   4   │ init-og      1                   root  txt       REG                8,1     14264    1972824 /usr/bin/init-og
   5   │ init-og      1                   root  mem       REG                8,1   2104984    1979669 /usr/lib/libc.so.6
   6   │ init-og      1                   root  mem       REG                8,1    228376    1978771 /usr/lib/ld-linux-x86-64.so.2
```

After glibc is upgraded, the files are still referenced as DEL(eted) as they have been replaced.
The init process has to be re-executed to "refresh" its references to the glibc files.

**Comment by valr** [comment]
Fixed by commit 2001e2467c664117c6c1e457ef2668bcaf5b3455
